### DISCUSSION TITLE
Créer un simulateur de salaire minimum (SMIC)

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2611] [Nouveaux simulateurs]
Nous ajoutons progressivement le droit conventionnel à notre base de règle, mais ne traitons pas le sujet important du salaire minimum. Le moteur doit pouvoir gérer cet aspect sans problème.

**Comment by loicguillois** [comment]
nouveau simulateur à part entière ou bien des évolutions fonctionnelles  / UX sur  le  simulateur salarié ?

**Comment by mquandalle** [comment]
Cf. https://github.com/SocialGouv/code-du-travail-numerique/issues/1403
